src/utils/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const PAGE_TITLE = 'The Guardian Browser';
+export const URL_BASE = 'https://content.guardianapis.com/';
+export const API_KEY = 'api-key';
```

Comment:
Remember about adding in README file of this project how and where to obtain api-key to test this app.